File: src/sdk-reference/js/6/realtime/publish/snippets/publish.js
const message = { realtime: 'rule the web' };

try {
  const published = await kuzzle.realtime.publish('i-dont-exist', 'in-database', message);

  if (published) {
    console.log('Success');
  }
} catch (error) {
  console.error(error.message);
}

File: src/sdk/java/1/controllers/realtime/publish/snippets/publish.java
try {
    String message = "{ \"realtime\": \"rule the web\" }";
    kuzzle.getRealtime().publish("i-dont-exist", "in-database", message);

    System.out.println("Success");
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}
